zppy/templates/e3sm_diags.bash

```diff
@@ -88,6 +90,21 @@ done
 cd ..
 {%- endif %}
 
+{%- if "streamflow" in sets %}
+# TODO: generalize "native"
+rofDir="{{ output }}/post/rof/native/ts/monthly/{{ ts_num_years }}yr"
+mkdir -p rof_links
+cd rof_links
+v="RIVER_DISCHARGE_OVER_LAND_LIQ"
+xml_name=${v}_${Y1}01_${Y2}12.xml
+cdscan -x ${xml_name} ${rofDir}/${v}_*.nc
```

Comment:
@chengzhuzhang @golaz I don't think this code block is sufficient to process data for Streamflow Diags. Am I missing a step?

If I use `/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/test/time-series/streamflow_ts_postprocessed/RIVER_DISCHARGE_OVER_LAND_LIQ_198001_201412.nc` (from the `complete_run` test data), I can get `zppy` to produce streamflow plots.

However, if I use the `xml` file generated by this block (e.g., `/lcrc/group/e3sm/ac.forsyth2/zppy_development/v2.LR.historical_0201/post/scripts/tmp.82707.ZeVE/rof_links/RIVER_DISCHARGE_OVER_LAND_LIQ_198501_201412.xml`), it fails.

Reply:
Hey @forsyth2, the RIVER_DISCHARGE_OVER_LAND_LIQ time series in the e3sm_diags folder include drainage area as an auxiliary field (time invariant). The preprocessing step is: 
```
cd ${drc_in}; ls *mosart.h0.20*nc | ncclimo --var_xtr=areatotal2 -v RIVER_DISCHARGE_OVER_LAND_LIQ --yr_srt=2000 --yr_end=2014 --drc_in=$drc_in --drc_out=$drc 
```
My guess is that the *.xml file does't have areatotal2 in there.

Reply:
@chengzhuzhang Thanks, the mistake turned out to be in my config file. I set `var_xtr` instead of `extra_vars = 'areatotal2'` in `[[ rof_monthly ]]`. However, I'm still running to an error in E3SM Diags:

```
  File "/home/ac.forsyth2/miniconda3/envs/e3sm_diags_env_dev_20210827/lib/python3.9/site-packages/e3sm_diags/driver/utils/dataset.py", line 183, in get_static_variable
    with cdms2.open(file_path) as f:
AttributeError: __enter__
```
However, I've confirmed that `file_path` is `rof_links/RIVER_DISCHARGE_OVER_LAND_LIQ_198501_201412.xml`, which does exist. I'm not sure why `cdms2.open` can't seem to open it.

Reply:
@forsyth2 I don't seem to have permission to the xml file, would you give access?

Reply:
@chengzhuzhang I just changed the permissions to `/lcrc/group/e3sm/ac.forsyth2/zppy_development/v2.LR.historical_0201/post/scripts/tmp.82796.ac45/rof_links/`, so you should be able to access.

Reply:
Thanks @forsyth2. It does seem the nc files linked by the xml file included areatotal2. I'm wondering if the particular function `get_static_variable` needs to be modified to support xml file with multiple variables...

Reply:
@chengzhuzhang That seems logical to me. I created https://github.com/E3SM-Project/e3sm_diags/issues/515 to address this.